packages/slate/src/create-editor.ts

```diff
@@ -217,7 +217,7 @@ export const createEditor = (): Editor => {
           (editor.isInline(node) ||
             node.children.length === 0 ||
             Text.isText(node.children[0]) ||
-            editor.isInline(node.children[0]))
+            editor.isInline(node.children[0] as Element))
```

Comment:
I think we should be staying away from type assertions, if at all possible.  There is a way to have the same code here w/o the type assertion.